packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -22,13 +25,10 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
 --alert__font-color: #262626;
 --alert__icon-size: 1.5rem;
 --alert__padding: 16px;
---animation-ease-in-out-expo: cubic-bezier(1, 0, 0, 1);
---animation-speed-1: 0.25s;
---animation-speed-2: 0.3s;
---animation-speed-3: 0.5s;
---animation-speed-4: 0.8s;
---animation-speed-base: 1;
---article-max-width: 600px;
```

Comment:
We don't use this variable and removed it

Reply:
are the `--animation-speed-*` vars used? i think i'm only seeing speed 2

Reply:
I'm seeing 1, 2, and 4 being used